.gitignore

```diff
@@ -3,6 +3,7 @@
 
 # Compiled Dirs
 public/photoBannerBundle.js
+public/styles.css
```

Comment:
Is there a reason why you added this file in .gitignore? I thought you had refactored and moved this file to the components directory.

Reply:
Removed. Thanks!